SETUP.md

```diff
@@ -0,0 +1,28 @@
+# Setup instructions
+
+Before you start using this template, complete the following steps:
+
+## Required changes
+
+- [ ] Protect the `main` branch
+- [ ] Add a short description of the repository
+- [ ] Add tags for the repository
+  - [ ] `open-source`,
+  - [ ] technology tag (`android`, `ios`, or `flutter`),
+  - [ ] other tags that are relevant to the repository
+- [ ] Create a `.gitignore` file
+- [ ] Create workflows
```

Comment:
For? This seems a bit vague, maybe they mean CI setup but I'm not sure if that is relevant to the template